source/common/orca/orca_load_metrics.h

```diff
@@ -0,0 +1,34 @@
+#pragma once
+
+#ifndef THIRD_PARTY_ENVOY_SRC_SOURCE_COMMON_ORCA_ORCA_LOAD_METRICS_H_
+#define THIRD_PARTY_ENVOY_SRC_SOURCE_COMMON_ORCA_ORCA_LOAD_METRICS_H_
```

Comment:
no need for these with the `#pragma once` above.

Reply:
Done